packages/google_maps_flutter/google_maps_flutter_ios/example/ios14/ios/Podfile

```diff
@@ -28,6 +28,9 @@ require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelpe
 flutter_ios_podfile_setup
 
 target 'Runner' do
+  use_frameworks!
+  use_modular_headers!
```

Comment:
I know we had some reservations about adding these to the examples before, but it's the only way I could get the `RunnerTest`s to compile

Reply:
Also new Flutter projects generate with these lines

Reply:
> Also new Flutter projects generate with these lines

Only Swift projects. Anyone with existing Obj-C projects won't have them.

Is it only RunnerTest, not Runner, that requires these? If it's only RunnerTest that's not a problem.